java-lib/src/main/avro/Reporting.avdl

```diff
@@ -64,6 +64,20 @@ protocol Reporting {
     union{null, string} spanSecondaryId = null;
     // span log entries
     array<SpanLog> logs;
+    // span in line data format, for sampling and filterings
+    union{null, string} span = null;
+  }
+
+  record SpanLogsDAO {
```

Comment:
Is this record really necessary?

Reply:
I think this is necessary for the data layer, but removed it because it doesn't really belong here.